curiefense/curieproxy/rust/curiefense/src/config/raw.rs

```
@@ -100,7 +100,7 @@ impl<'de> Deserialize<'de> for RawGlobalFilterSSectionEntry {
                 let tp = seq.next_element()?.ok_or_else(|| de::Error::invalid_length(0, &self))?;
                 let vl = seq.next_element()?.ok_or_else(|| de::Error::invalid_length(1, &self))?;
                 // comment might not be present
-                let comment = seq.next_element()?;
+                let comment = seq.next_element().ok().flatten();
```

I'm a bit unsure whether discarding the error is the best idea here.